app/views/partials/foyer/patrimoine.html

```diff
@@ -140,11 +136,11 @@ <h2 class="panel-title">
         </div>
 
         <div class="form-group">
-          <label class="col-sm-4 control-label">Détenez-vous un <abbr title="Plan Épargne Logement">PEL</abbr>, un <abbr title="Plan d'épargne retraite populaire">PERP</abbr>, des titres, une assurance-vie, des parts d'une <abbr title="Société d'Investissement à CApital Variable">SICAV</abbr> ou d'un <abbr title="Fonds Commun de Placement">FCP</abbr> ?</label>
```

Comment:
Good catch! This indeed broke the layout.

As in reviewed code
![capture d ecran 2016-08-12 a 10 51 49](https://cloud.githubusercontent.com/assets/222463/17617579/3c099980-607b-11e6-9c99-a19c724760b7.png)

After removing the unnecessary `form-group`:
![capture d ecran 2016-08-12 a 10 52 48](https://cloud.githubusercontent.com/assets/222463/17617583/3f7e93ae-607b-11e6-914e-4828f1c1aebf.png)